src/osp/Resource/AssetImporter.cpp

```diff
@@ -469,8 +469,8 @@ void AssetImporter::proto_add_obj_recurse(
         // TODO: Add support different collider shapes here!
         std::string const& shapeName = extras.Get("shape").Get<std::string>();
         // change this to some map too
-        const phys::ECollisionShape shape = (shapeName == "cylinder")
-            ? phys::ECollisionShape::CYLINDER : phys::ECollisionShape::BOX;
+        const phys::EShape shape = (shapeName == "cylinder")
+            ? phys::EShape::Cylinder : phys::EShape::Box;
```

Comment:
personally i always put the ? and : on different lines, but it's not a big deal.